### PR TITLE
[OO-BUGFIX] Improve mime-type detection reliability on media upload

### PR DESCRIPTION
**Pull request by Thiblef**

### CODE REVIEW

File: MediaAdminBundle/Manager/SaveMediaManager.php [hunk 1]
```diff
@@ -135,7 +148,7 @@ public function saveMedia($media)
     protected function processMedia($media, $uploadedFile, $filename)
     {
         $media->setName($uploadedFile->getClientOriginalName());
-        $media->setMimeType($uploadedFile->getClientMimeType());
+        $media->setMimeType($this->getMimeType($filename));
```
Comment (alavieille) [inline MediaAdminBundle/Manager/SaveMediaManager.php:151]: i think the method getMimeType of class UploadedFile doing already the job ?

Comment (Thiblef) [inline MediaAdminBundle/Manager/SaveMediaManager.php:151]: I tried it `$uploadedFile->getMimeType()` but it throwed a file not found exception, don't know why.
Moreover you already use the `finfo_file` method on the filename in the `MediaController` :
`if ($saveMediaManager->isFileAllowed($filename))`.

**Comment by ngilain** [comment]
@alavieille , @Thiblef : some news on this one ?

**Comment by alavieille** [comment]
These modifications are merged in this PR https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/284 thanks

**Pull request closed without merging**